js/ui/codingManager.js

```diff
@@ -473,9 +473,14 @@ const CodingManager = new Lang.Class({
     },
 
     _overviewHiding: function(overview, session) {
-        session.buttonApp.show();
-        if (session.buttonBuilder)
+        let focusedWindow = global.display.get_focus_window();
+        if (!focusedWindow)
+            return;
+
+        if (session.actorBuilder && session.actorBuilder.meta_window === focusedWindow)
             session.buttonBuilder.show();
+        else if (session.actorApp.meta_window === focusedWindow)
```

Comment:
is `session.actorApp` nullable like `session.actorBuilder` above? If not, please add a comment.